src/settings-ui/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -962,4 +962,7 @@
   <data name="Run_NotAllowedActionKeyword.Text" xml:space="preserve">
     <value>This activation phrase overrides the behavior of other plugins. Please change it to something else.</value>
   </data>
-</root>
+  <data name="Run_PluginUseDescription.Text" xml:space="preserve">
+    <value>You can include or remove each plugin from the global results, change the  direct activation phrase and configure additional options.</value>

```

Comment:
@enricogior
This description doesn't indicate that the user has to click the plugin entry. But your initial concern was to hint the user that they has to click the plugin.

Reply:
What is important is to give a hint that there is more for each plugin to configure.
It's a step in the right direction in terms of discoverability.